lighthouse-core/test/audits/installable-manifest-test.js

```diff
@@ -167,7 +167,7 @@ describe('PWA: webapp install banner audit', () => {
       return InstallableManifestAudit.audit(artifacts, context).then(result => {
         assert.strictEqual(result.score, 0);
         const items = result.details.items;
-        expect(items[0].reason).toBeDisplayString(/unexpected arguments/);
+        expect(items[0].reason).toMatch(/unexpected arguments/);
```

Comment:
these two are kind of weird because we're putting them in the audit table, but really they're messages to us that the protocol has diverged from what we expect, so it seems better to leave them untranslated (maybe they should be Sentry errors and test assertions that they don't occur in CI?)